.changeset/sharp-lions-cover.md

```diff
@@ -0,0 +1,7 @@
+---
+'@wallet-standard/errors': major
+'@wallet-standard/react-core': major
+'@wallet-standard/core': major
```

Comment:
Is this changeset still correct? It looks like `react-core` isn't part of this PR and I'm not sure if the new `ui` package needs to be in the changeset

Reply:
Oof. Yeah. I'll reword all of these.